io.home-assistant.binary-sensor.gas/dataset.tt

```diff
@@ -0,0 +1,128 @@
+dataset @io.home-assistant.binary-sensor.gas language "en" {
+
+  program := now => @io.home-assistant.binary-sensor.gas.state() => notify
+  #_[utterances=["what is the state of my gas sensor?",
+                "what is my gas sensor showing?",
+                "what does my gas sensor say?",
+                "is my gas sensor detecting gas?",
+                "check if my gas sensor is detecting gas",
+                "is my gas sensor detecting any gas?",
+                "check if my gas sensor is detecting any gas",
```

Comment:
A lot of these utterances feel weird. People don't often say the word "sensor", instead people refer to what it detects.

"is gas detected?" , "check if there is gas" etc.